README.md

```diff
@@ -6,7 +6,7 @@ A Docker Development Build and Test Container where Projects are hammered into s
 
 ![Docker Pulls](https://img.shields.io/docker/pulls/axrs/anvil?style=for-the-badge)
 
-Base Image [Debian:bullseye-slim](https://hub.docker.com/_/debian)
+Base Image [Debian:stable-slim](https://hub.docker.com/_/debian). See [Debian Releases](https://www.debian.org/releases/)
```

Comment:
Sable is bullseye ATM

Reply:
Stable is a tag that changes over time. These containers should be stable in that a major OS bump shouldn't just occur. Especially if users have setup package repos that target bullseye specifically.